dbase/interpreter.go

```diff
@@ -90,21 +92,212 @@ func (dbf *DBF) DataToValue(raw []byte, column *Column) (interface{}, error) {
 	case "N":
 		// N values are stored as string values, if no decimals return as int64, if decimals treat as float64
 		if column.Decimals == 0 {
-			i, err := dbf.parseNumericInt(raw)
+			i, err := parseNumericInt(raw)
 			if err != nil {
-				return i, fmt.Errorf("dbase-interpreter-datatovalue-7:FAILED:%w", err)
+				return i, fmt.Errorf("dbase-interpreter-datatovalue-7:FAILED:parsing numeric int at column field: %v failed with error: %w", column.Name(), err)
 			}
 			return i, nil
 		}
 		fallthrough // same as "F"
 	case "F":
 		// F values are stored as string values
-		f, err := dbf.parseFloat(raw)
+		f, err := parseFloat(raw)
 		if err != nil {
-			return f, fmt.Errorf("dbase-interpreter-datatovalue-8:FAILED:%w", err)
+			return f, fmt.Errorf("dbase-interpreter-datatovalue-8:FAILED:parsing float at column field: %v failed with error: %w", column.Name(), err)
 		}
 		return f, nil
 	default:
 		return nil, fmt.Errorf("dbase-interpreter-datatovalue-9:FAILED:Unsupported column data type: %s", column.Type())
 	}
 }
+
+// Converts column data to the byte representation
+// For M values the data has to be written to the memo file
+func (dbf *DBF) valueToData(field *Field) ([]byte, error) {
```

Comment:
In my opinion, valueToData does not correctly describe what this function actually does. Maybe something like "valueToByteRepresentation" or "valueToByteArray" would be more descriptive.